utils/docker/Dockerfile.noble

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:noble
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG TZ=America/Los_Angeles
+ARG DOCKER_IMAGE_NAME_TEMPLATE="mcr.microsoft.com/playwright/java:v%version%-noble"
+
+# === INSTALL JDK and Maven ===
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends openjdk-21-jdk \
+    # Install utilities required for downloading browsers
+    curl \
+    # Install utilities required for downloading driver
+    unzip \
+    # For the MSEdge install script
+    gpg && \
+    rm -rf /var/lib/apt/lists/* && \
+    # Create the pwuser
+    adduser pwuser
+
+# Ubuntu 22.04 and earlier come with Maven 3.6.3 which fails with
```

Comment:
Is this relevant to 24.04? Let's double check?

Reply:
Let's check this

Reply:
As per offline discussion with Yury:

> The maven version we distribute is never (3.8.7 vs. 3.9.x) than what users get in Ubuntu 24.04 via apt.
I think its better to have it manually installed like we do, then its aligned in all our 3 images, wdyt?